src/views/missingScores.tmpl

```diff
@@ -8,27 +8,27 @@
 <ul>
     <li>
         <span class="stat-description">Total max scores:</span>
-        {{.NumMaxScores}} / {{.TotalMaxScores}}
+        {{.NumMaxScores}}
```

Comment:
This could also be made out of the total number of games to add additional context.